lib/trailblazer/operation/controller.rb

```diff
@@ -64,8 +64,9 @@ def operation!(operation_class, params)
       # this is what happens:
       # respond_with Comment::Update::JSON.run(params.merge(comment: request.body.string))
       concept_name = operation_class.model_class.to_s.underscore # this could be renamed to ::concept_class soon.
+      request_body = request.body.respond_to?(:string) ? request.body.string : request.body.read
```

Comment:
I usually _hate_ introspective code, but here, it really has to be. What if we use `read` for all? Maybe Rails tests will work with this, now? It's the official public method for a Rack request.